addons/bottom-navigation-bar/addon.json

```
@@ -0,0 +1,27 @@
+{
+  "name": "Bottom navigation bar",
+  "description": "Positions the navigation bar at the bottom of the page.",
+  "tags": ["community", "theme"],
+  "versionAdded": "1.27.2",

```

```suggestion
  "versionAdded": "1.28.0",

```
We are on 1.28.0-pre.